docs/source/releases/latest/698-install-packages-prior-to-attempting-to-build-their-documentation.yaml

```diff
@@ -0,0 +1,112 @@
+bug fix:
+- description: |
+    Add a step in the ``build_sphinx_html`` CI job to install the package for which we
+    want to build documentation prior to building the documentation. Also remove the
+    installation code from the ``build_docs.sh`` script.
+
+    This fixes an issue where installation doesn't always happen correctly causing the
+    docs to fail to build.
+  files:
+    added:
+    - ''
+    deleted:
+    - ''
+    modified:
+    - 'docs/build_docs.sh'
+    - '.github/workflows/doc-lint-test.yaml'
+    moved:
+    - ''
+  related-issue:
+    number: 0
+    repo_url: ''
+  title: 'Install packages prior to attempting to build their documentation'
```

Comment:
@biosafetylvl5  Does brassy ignore all the empty ones?  Or do we end up with a bunch of empty sections in latest.rst?

Reply:
Jeremy said it was fine :)

Reply:
It should!